### DISCUSSION TITLE
Start Generator Button Disappeared.

### DISCUSSION THREAD

**Discussion by Mayoroptic** [jgyates/genmon#1135] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Version

1.19.03

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have submitted logs

### What is/are your question(s)?

Hello I am just reaching out in regard to the Start Generator Button. 

I had started and stopped the generator a few times manually with this button, but now the button no longer seems to show up on the Genmon Site (I think maintenance was the right tab)

![image](https://github.com/jgyates/genmon/assets/170766631/fb9d3922-2e68-4188-8188-33be7b299698)

You can see here that the gauges do still work. And the scheduled exercise works as well.

![image](https://github.com/jgyates/genmon/assets/170766631/13de5ca4-1fbc-4754-a3f6-4ddb1177824f)

### Additional information

Device: Raspberry Pi Zero 2 W with HAT from printsizeme
OS: PRETTY_NAME="Raspbian GNU/Linux 11 (bullseye)"
NAME="Raspbian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"

**Comment by jgyates** [comment]
I have not received any logs. To send logs you must have outbound email enabled within genmon.

Have you tried refreshing your browser?
Did you enable the "Smart Switch" option on the setting page? Enabling this will disable the remote start, stop, start/transfer buttons.

Other than that suggestion I would need to see your logs to give any other suggestions.

**Reply by jgyates** [reply, marked as answer]
The setting mentioned above is "Smart Transfer Switch" on the settings page. if you enable this you can not use remote functions. Most people do not have a smart transfer switch. A Smart Switch will monitor the utility voltage instead of the controller.

**Reply by Mayoroptic** [reply]
@jgyates This was the answer! Thank you for the follow up.
